src/data-workspace/custom-form/custom-form.module.css

```diff
@@ -1,5 +1,5 @@
 .customForm {
-    width: 100%;
+    inline-size: 100%;
```

Comment:
these css updates are just formatting fixes (because of upgraded d2-style). They just got picked up when I ran `yarn format` but are not related to the ticket